addons/website_forum/controllers/main.py

```diff
@@ -553,7 +553,10 @@ def _prepare_user_profile_values(self, user, **post):
                 forums = post['forum']
             elif post.get('forum_id'):
                 forums = request.env['forum.forum'].browse(int(post['forum_id']))
-                values.update({'edit_button_url_param': 'forum_id=' + str(post['forum_id'])})
+                values.update({
+                    'edit_button_url_param': 'forum_id=' + str(post['forum_id']),
```

Comment:
While we are touching this line, maybe remove string concatenation and replace it with a real formatting (forum_id=%s % blah).